simhash/__init__.py

```diff
@@ -168,6 +168,22 @@ def distance(self, another):
         return ans
 
 
+class MultiSimhash(Simhash):
+    def __init__(self, simhashes):
+        multi_f = 0
+        if not isinstance(simhashes, Iterable):
+            raise Exception('Value passed is not a list of simhashes')
+        for i in simhashes:
+            multi_f = multi_f + i.f
+        if multi_f % 8:
+            raise Exception('Simhashes do not the same length (f)')
```

Comment:
Hi, could you explain here, what do you want to check?

Reply:
Hi, well looking at it twice, that code doesn't make much sense. I will change it to check them all 1 by 1 to the length of the first simshash.